doc/lvm.md

```diff
@@ -0,0 +1,168 @@
+# LVM features and YaST
+
+The basic LVM functionality has been supported by YaST for ages. But LVM has a lot of features
+like RAID, snapshots, thin provisioning and whatsnot that often result in special types of logical
+volumes being present in the system.
+
+This document summarizes how well can YaST deal with systems that contain such advanced LVM setups.
+
+## The Partitioner
+
+This section describes how the Expert Partitioner represent the different LVM technologies and what
+operations it allows for each type of logical volume. At the current stage, some operations show an
+unexpected behavior and, in most cases, would need to be adjusted. That is represented in bold text.
+
+Note that, unlike RAID0, stripped LVs are not really a separate type. Many types of LVs can be
+stripped.
+
+### Normal LVM
+
+This is somehow the base case. Normal LVs are visualized in the standard way and all operations
+are supported.
+
+### Thin Pool and Thin LV
+
+#### How is it displayed?
+
+- Only the thin pool and its thin LVs are displayed. No trace of the hidden LVs used to store the
+  data and the metadata of each thin LV or to store the spare metadata of the VG.
+- Due to a bug, **nothing in the UI identifies the displayed LVs as being special**. They basically
+  look like normal LVs, although `BlkDevicesTable::DEVICE_LABELS` contains entries for both thin
+  pools and thin LVs.
+- In LVM is not possible to define stripping for thin LVs, they use the stripping defined for their thin
+  pools. The partitioner UI **reports 0 stripes for all thin LVs**.
+
+#### What can be done?
+
+- For thin pools
+  - Create: it works.
+  - Edit (format/mount): not allowed ("_Is a thin pool. It cannot be edited_").
+  - Resize: it works. Thin pools already in disk cannot be shrinked, which is correct.
+  - Delete: it works. Note it deletes the pool, all its thin volumes and the associated hidden LVs.
+
+- For thin LVs
+  - Create: it works. The **widgets for defining stripping are disabled and set to the default values**.
+    Maybe it would be better to show them disabled but with the pool values. Or to not show them at all.
+  - Edit (format/mount): not allowed ("_Is a thin pool. It cannot be edited_").
```

Comment:
If that message is shows it is a serious bug.

Reply:
Why? Not allowing to format/mount thin pools makes sense. Isn't it?

Reply:
Yes, but it in the context of thin logical volumes.

Reply:
Oops, copy&paste error. I will fix it.

Reply:
Fixed.